src/index.tsx

```diff
@@ -277,8 +277,9 @@ class ReactQuill extends React.Component<ReactQuillProps, ReactQuillState> {
       const {delta, selection} = this.regenerationSnapshot!;
       delete this.regenerationSnapshot;
       this.instantiateEditor();
-      this.editor!.setContents(delta);
-      if (selection) postpone(() => this.editor!.setSelection(selection));
+      const editor = this.editor!;
+      editor.setContents(delta);
+      if (selection) postpone(() => editor.setSelection(selection));
```

Comment:
I think this can be converted to use `this.setEditorSelection` with no particular ill effects. Can you verify?

Reply:
It works fine for my usecase.

What was the reason for using postpone here? I wonder if it's possible for the selection to "jump back" if there is text inserted between the time setContents is called and the time the selection is applied. Although that race seems unlikely 🤔

Reply:
> What was the reason for using postpone here? 

Controlled-mode Quill blurs the editor in the micro-tick immediately after `setContents`. Postponing this code allows overriding that blur. I wasn't able to find a deeper cause yet, and this solved the issue for controlled mode.

> I wonder if it's possible for the selection to "jump back" if there is text inserted between the time setContents is called and the time the selection is applied. Although that race seems unlikely 🤔
 
The race is unlikely as it happens in the very next micro-tick, and is queued before user code has a chance to run.

If `postpone -> this.setEditorSelection` works, let's use that, and unify the approaches. Worst case we will have a reproduction case showing us why not.

Other than this, it can get merged for me 👍 thanks a lot for the excellent work